hummingbot/connector/exchange/loopring/loopring_exchange.pyx

```diff
@@ -44,7 +44,7 @@ from hummingbot.core.event.events import (
     TradeType,
     OrderType,
 )
-from hummingbot.core.data_type.trade_fee import TradeFee
+from hummingbot.core.data_type.trade_fee import AddedToCostTradeFee
```

Comment:
`hummingbot.core` should come after. And `hummingbot.logger` is in the wrong place too.